docker/build_scripts/build.sh

```diff
@@ -46,15 +46,14 @@ echo "multilib_policy=best" >> /etc/yum.conf
 # Decided not to clean at this point: https://github.com/pypa/manylinux/pull/129
 yum -y update
 
-# Software collection (for devtoolset-8) and EPEL support (for cmake28 & yasm)
-yum -y install centos-release-scl https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
+# Software collection (for devtoolset-8) and EPEL support (for yasm)
```

Comment:
Worth adding "export LANG=C LC_ALL=C". Otherwise build log will be full of 
```
/bin/sh: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
```
messages

Reply:
resolved in my aarch64 branch, in the Dockerfile.